packages/client/tests/functional/composites-mongo-list/count.ts

```diff
@@ -0,0 +1,50 @@
+import { faker } from '@faker-js/faker'
+import pRetry from 'p-retry'
+
+import { setupTestSuite } from './_matrix'
+
+// @ts-ignore this is just for type checks
+declare let prisma: import('@prisma/client').PrismaClient
+
+const id = faker.database.mongodbObjectId()
+
+setupTestSuite(() => {
+  beforeAll(async () => {
+    await pRetry(
```

Comment:
@Jolg42 do you think we still need those retries for `create` considering that each suite starts will clean DB and we use randomized ids so we never have to clean the database between different tests?

Reply:
I think the database isolation does not work correctly at the moment, and for that reason, the amount of workers is set to `1` in the jest config. This also means it is safe to remove the retry logic, but it also means that we need to fix the isolation issues.

Reply:
Thanks everyone, I'll keep retries for now and re-check it once we fix DB isolation.